lib/experimental/Navigation/Header/Breadcrumbs/index.tsx

```diff
@@ -34,32 +73,25 @@ interface BreadcrumbItemProps {
 type DropdownItemWithoutIcon = Omit<DropdownItemObject, "icon">
 
 function BreadcrumbItem({ item, isLast }: BreadcrumbItemProps) {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const { label, ...props } = item
+  const isLoading = "loading" in item
+  const content = isLast ? (
```

Comment:
It is not clear and together with chained ternary operators have also nested ones. Can you extract it to a component with a simpler control flow?

Reply:
Yeah, I think I will. I just found out of this scenario by accident, there's a bit of coupling. I'm thinking that the "embedded" scenario should be dealt with from the breadcrumbs directly, but maybe that's for another PR.